pkg/kudoctl/env/environment.go

```diff
@@ -32,42 +49,14 @@ var DefaultSettings = &Settings{
 	RequestTimeout: 0,
 }
 
-// envMap maps flag names to envvars
-var envMap = map[string]string{
-	"home":       "KUDO_HOME",
-	"kubeconfig": "KUBECONFIG",
-}
-
 // AddFlags binds flags to the given flagset.
 func (s *Settings) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar((*string)(&s.Home), "home", DefaultKudoHome, "location of your KUDO config.")
-	fs.StringVar(&s.KubeConfig, "kubeconfig", os.Getenv("HOME")+"/.kube/config", "Path to your Kubernetes configuration file.")
```

Comment:
`kubeconfig` location on `GOOS == "windows"` was broken since there is no `os.Getenv("HOME")` there.